lib/ansible/galaxy/collection/__init__.py

```diff
@@ -769,6 +769,9 @@ def install_collections(
                     "Skipping signature verification."
                 )
 
+            if concrete_coll_pin.type == 'galaxy':
+                concrete_coll_pin = concrete_coll_pin.with_signatures_repopulated(concrete_coll_pin)
```

Comment:
If this were to be used as a true classmethod, it'd be called as `Candidate.with_repopulated_signatures_extending(concrete_coll_pin)`. What I originally meant was just:
```suggestion
                concrete_coll_pin = concrete_coll_pin.with_signatures_repopulated()
```

Reply:
Thanks for reviewing again. Fixed this.